### DISCUSSION TITLE
sunsynk-power-flow-card configuration

### DISCUSSION THREAD

**Discussion by Cyrelion** [marq24/ha-senec-v3#32] [Show and tell]
Hallöchen,

hat jemand schon mal ein gutes Mapping der Senec Werte in der SunSynk frontend-Card (https://github.com/slipx06/sunsynk-power-flow-card) hinbekommen?

Aktuell bin ich schon relativ weit, nur ein paar Sachen passen noch nicht so ganz.

type: custom:sunsynk-power-flow-card
cardstyle: full
show_solar: true
battery:
  energy: 10000
  shutdown_soc: 20
  show_daily: true
  max_power: 3700
  invert_power: true
solar:
  show_daily: true
  mppts: 2
  pv1_name: Senec
  pv2_name: SMA
  auto_scale: true
load:
  show_daily: true
grid:
  show_daily_buy: true
  show_daily_sell: true
  show_nonessential: false
entities:
  use_timer_248: false
  priority_load_243: switch.senec_safe_charge
  inverter_voltage_154: false
  load_frequency_192: sensor.senec_enfluri_net_freq
  inverter_current_164: false
  inverter_power_175: sensor.senec_house_power
  grid_connected_status_194: false
  inverter_status_59: sensor.senec_system_state
  day_battery_charge_70: sensor.senec_battery_charged_daily
  day_battery_discharge_71: sensor.senec_battery_discharged_daily
  battery_soc_184: sensor.senec_battery_charge_percent
  battery_power_190: sensor.senec_battery_state_power
  battery_current_191: false
  grid_power_169: sensor.electricitymeter_total_power
  day_grid_import_76: sensor.electricitymeter_imported_daily
  day_grid_export_77: sensor.electricitymeter_exported_daily
  grid_ct_power_172: sensor.electricitymeter_total_power
  day_load_energy_84: sensor.senec_house_consumed_daily
  essential_power: sensor.senec_house_power
  nonessential_power: none
  aux_power_166: sensor.sunsynk_aux_power
  day_pv_energy_108: sensor.senec_solar_generated_daily
  pv1_power_186: sensor.senec_solar_mpp_power
  pv2_power_187: sensor.sn_3015762752_pv_power
  pv1_voltage_109: false
  pv1_current_110: false

**Comment by marq24** [comment]
with the latest 'beta' you could get the battery_current_191: senec.battery_state_current - and when using the Inverter Integration you will have for sure access to the 'inverter_current_164' as well
